docs/ref/contrib/syndication.txt

```diff
@@ -338,9 +340,10 @@ to something different. Then update your URLconf to add the extra versions.
 Here's a full example::
 
     from django.contrib.syndication.views import Feed
-    from policebeat.models import NewsItem
     from django.utils.feedgenerator import Atom1Feed
 
+    from .models import NewsItem
```

Comment:
Leave alone as above. The next example implies this file is `myproject/feeds.py`.

Reply:
ok